analysis/data_gen_utils.py

```diff
@@ -539,8 +522,11 @@ def download_IBL(pid, save_folder, t_window=[0, 500]):
             sr.file_meta_data,
             metadata_file,
         )
-
-        return standardized_file, metadata_file
+        print("done with destriping")
+    metadata_file = standardized_file.parent.joinpath(
```

Comment:
can move this line to before the if conditional and remove the metadata_file definition inside the if conditional (line 517)